src/nft/components/bag/BagHeader.tsx

```diff
@@ -56,19 +72,17 @@ interface BagHeaderProps {
 
 export const BagHeader = ({ numberOfAssets, closeBag, resetFlow, isProfilePage }: BagHeaderProps) => {
   return (
-    <Column gap="4" paddingX="32" marginBottom="20">
-      <Row className={styles.header}>
-        <ThemedText.HeadlineSmall>{isProfilePage ? 'Sell NFTs' : 'My bag'}</ThemedText.HeadlineSmall>
-        <IconWrapper onClick={closeBag}>
-          <BagCloseIcon />
-        </IconWrapper>
-      </Row>
+    <Wrapper>
+      <ThemedText.HeadlineSmall>{isProfilePage ? <Trans>Sell</Trans> : <Trans>Bag</Trans>}</ThemedText.HeadlineSmall>
       {numberOfAssets > 0 && (
-        <ControlRow>
-          <ThemedText.BodyPrimary>{roundAndPluralize(numberOfAssets, 'NFT')}</ThemedText.BodyPrimary>
+        <>
+          <NumberDot>{numberOfAssets}</NumberDot>
```

Comment:
Can you wrap numberOfAssets in its own div, and set padding to 4px on the left and right.

Setting a fixed height and width in NumberDot leads to situations like this if the number is very long: 
<img width="170" alt="Screen Shot 2022-11-14 at 12 27 06 PM" src="https://user-images.githubusercontent.com/41491154/201726352-580cc7d8-2d3a-4a5b-9c05-eef50e5f5462.png">

Reply:
Also in figma the NumberDot element is called CounterDot, might be helpful to rename for consistency with future uses of this component in Figma / designs.

Reply:
Out max bag size is 100, so the most we can expect is triple digits here 
<img width="554" alt="image" src="https://user-images.githubusercontent.com/5773490/201728408-3005a150-8762-49be-9f93-6e6297e15e92.png">

Reply:
yeah, the pr now does that--just took this pic before I'd made that change (removing the fixed width).